internal/ssl/.ssl/fullchain.pem

```diff
@@ -0,0 +1,90 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
are these "good" certificates? I'd suggest using dummy certs here and generate good ones on the fly at build time in the CI.

Reply:
Yeah, this one is real. I'll replace it with self-signed and provide a simple shell script that generates a real one for internal usage and testing.